cli/tests/integration/npm_tests.rs

```diff
@@ -266,6 +266,117 @@ fn cached_only_after_first_run() {
   assert_contains!(stdout, "createChalk: chalk");
 }
 
+#[test]
+fn reload_flag() {
+  let _server = http_server();
+
+  let deno_dir = util::new_deno_dir();
+
+  let deno = util::deno_cmd_with_deno_dir(&deno_dir)
+    .current_dir(util::testdata_path())
+    .arg("run")
+    .arg("--unstable")
+    .arg("--allow-read")
+    .arg("--allow-env")
+    .arg("npm/reload/main.ts")
+    .env("NO_COLOR", "1")
+    .envs(env_vars())
+    .stdout(Stdio::piped())
+    .stderr(Stdio::piped())
+    .spawn()
+    .unwrap();
+  let output = deno.wait_with_output().unwrap();
+  let stderr = String::from_utf8_lossy(&output.stderr);
+  let stdout = String::from_utf8_lossy(&output.stdout);
+  assert_contains!(stderr, "Download");
+  assert_contains!(stdout, "createChalk: chalk");
+  assert!(output.status.success());
```

Comment:
We need to create a less verbose api for doing this kind of testing in the future.

Reply:
I agree, I'll see how many tests could use a helper